src/Makefile

```diff
@@ -36,8 +54,12 @@ distclean: clean
 	$(MAKE) $(MAKEFLAGS) -C components distclean
 	@rm -f `pwd`/../transfer/*
 
-init:
+init: | ensure_config init_extern init_composite
```

Comment:
Never used `|` in such a rule before. I'll need to look up what that does.

Reply:
It sequences the dependencies